test/unit/presubmit-tests.sh

```diff
@@ -270,4 +270,9 @@ test_function ${SUCCESS} "BUILD TESTS PASSED" main --build-tests
 test_function ${SUCCESS} "EXECUTING default_build_test_runner" main --build-tests
 test_function ${SUCCESS} "BUILD TESTS PASSED" call_function_pre run_markdown_build_tests
 
+echo ">> Testing custom and multi-script execution"
+
+test_function ${SUCCESS} "One" main --run-test "echo One"
+test_function ${SUCCESS} "Second" main --run-test "echo First" --run-test "echo Second"
```

Comment:
Any chance matching `First\nSecond`?

Reply:
I tried that, but it didn't work. From my research, it seems it only checks the last line so it will be hard to verify the correct execution of the first (unless we do something interesting such as piping from the first to the second and verify completeness)

Reply:
Haven't tried, just curious if this could work:
```
test_function ${SUCCESS} "First" main --run-test "echo First" --run-test "echo Second"
```
If not then we're out of luck for now

Reply:
Sorry for not making it clear, I meant add another test case like that